py3-exceptiongroup.yaml

```diff
@@ -25,9 +25,10 @@ vars:
 data:
   - name: py-versions
     items:
-      3.10: "310"
-      3.11: "311"
-      3.12: "312"
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '300'
```

Comment:
why not 313?  IDK why I didn't notice that before.

Reply:
@xnox saved us there.

because if it is 313 then 'apk add py3-exceptiongroup' will get you python 3.13, and we don't want that now.  The first reason we don't want it is that 3.13 isn't released, so we don't want it as "default".

Then we have other "does'nt build yet" reasons.

Reply:
Ahh that is a good check.  I'd rather we just not produce the unversioned packages and make people specify the version tbh.